config/grafana/status.json

```diff
@@ -32,6 +32,66 @@
       "title": "Kyma State Overview",
       "type": "row"
     },
+    {
+      "datasource": null,
+      "description": "This panel provides information on Which modules are enabled on the KCP and the number of their respective instances.",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 250
+              }
```

Comment:
Any specific Reason why the graph should switch to the color red if there are more than 250 modules enabled?

Lets all have them at the same color. Red normally shows that something goes into a direction which we do not want to have.

Reply:
The initial idea was to have the whole count category as red, if even a single instance of that module is in error state or not in the desired state. But I left it, since it was out of the scope of this issue. So let's just keep everything as green. :)